handlers/get.go

```diff
@@ -26,6 +26,10 @@ func (achievementHandler *AchievementsHandler) GetAchievementByID(responseWriter
 	achievement, err := data.GetAchievementByID(id)
 	switch err {
 	case nil:
+		err = data.ToJSON(achievement, responseWriter)
```

Comment:
Remove ToJSON and FromJSON and replace with our current way of doing it

Reply:
Tu avait déjà accepté cette PR, jecroyais qu'on aurait une tache de rafactor pour mettre dans les pkg, mais je peux le changer